src/tray.ts

```diff
@@ -38,6 +40,15 @@ const generateMenu = (
   const akashi = generateMenuItem('normal', 'AKASHI', () => {
     shell.openExternal('https://atnd.ak4.jp/login');
   });
+  const aboutDakokuApp = generateMenuItem('normal', 'About DakokuApp', () => {
```

Comment:
他が日本語なので 「DakokuApp について」ですかね
```suggestion
  const aboutDakokuApp = generateMenuItem('normal', 'DakokuApp について', () => {
```